templates/web/fixmystreet.com/report/new/roads_message.html

```diff
@@ -10,7 +10,7 @@
         other side of the border.
         </p>
     </div>
-    <div id="js-not-a-road" class="hidden js-responsibility-message js-roads-bucks js-roads-centralbeds js-roads-shropshire">
+    <div id="js-not-a-road" class="hidden js-responsibility-message js-roads-bucks js-roads-centralbeds js-roads-shropshire js-roads-camden">
```

Comment:
Doesn't really matter, but note the reason the other councils here have this class (and matching no_asset_msgs_class set in the config) is because they have multiple 'road' error templates here, all of which need hiding when one is shown (so it doesn't end up showing two), the normal ID based one isn't enough. So Camden doesn't actually need this class or the no_asset_msgs_class entry in the config, as it only has the one road error template that could be shown.